storage/aptosdb/src/db_options.rs

```diff
@@ -43,9 +43,10 @@ pub(super) fn state_merkle_db_column_families() -> Vec<ColumnFamilyName> {
     ]
 }
 
-pub(super) fn kv_db_column_families() -> Vec<ColumnFamilyName> {
+pub(super) fn state_kv_db_column_families() -> Vec<ColumnFamilyName> {
     vec![
         /* empty cf */ DEFAULT_COLUMN_FAMILY_NAME,
+        DB_METADATA_CF_NAME,
```

Comment:
why do we store DB metadata in every CF? Hav we consider a separate CF for metadata ?

Reply:
It is a separate CF.